src/main/fc/cleanflight_fc.c

```diff
@@ -696,6 +696,11 @@ void taskMainPidLoop(void)
 #endif
 
 #if defined(BARO) || defined(SONAR)
+        // FIXME outdates comments?
```

Comment:
yes, 100% agreed.  @ledvinap recently noted that the data flow is essentially wrong and dates all the way back to the multiwii codebase.